misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/src/test/resources/de/metas/camel/externalsystems/shopware6/happyFlow/50_CamelUpsertBPartnerCompositeRequest.json

```diff
@@ -3,7 +3,7 @@
   "jsonRequestBPartnerUpsert": {
     "requestItems": [
       {
-        "bpartnerIdentifier": "ext-Shopware6-customerId",
```

Comment:
please remove the customer-specific name

Reply:
removed.